pkg/cloud/aws/aws.go

```diff
@@ -191,13 +191,29 @@ func (c *AwsCloud) DeleteSecurityGroupRule(groupID, direction, protocol, ip stri
 }
 
 // CreateEC2Instances creates EC2 instances
-func (c *AwsCloud) CreateEC2Instances(prefix string, count int, amiID, instanceType, keyName, securityGroupID string, forMonitoring bool) ([]string, error) {
+func (c *AwsCloud) CreateEC2Instances(prefix string, count int, amiID, instanceType, keyName, securityGroupID string, forMonitoring bool, iops, throughput int) ([]string, error) {
 	var diskVolumeSize int32
 	if forMonitoring {
 		diskVolumeSize = constants.MonitoringCloudServerStorageSize
 	} else {
 		diskVolumeSize = constants.CloudServerStorageSize
 	}
+	iopsValue := int32(3000)      // 3000 is default for gp3
+	throughputValue := int32(125) // 125 is default for gp3
+	if iops != 0 {
+		iopsValue = int32(iops)
+	}
+	if throughput != 0 {
+		throughputValue = int32(throughput)
+	}
+	ebsValue := &types.EbsBlockDevice{
+		VolumeSize:          aws.Int32(diskVolumeSize),
+		VolumeType:          types.VolumeTypeGp3,
+		DeleteOnTermination: aws.Bool(true),
+		Throughput:          aws.Int32(throughputValue),
```

Comment:
I believe this only applies to gp3, so if you specify for other volume types you should error

Reply:
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2/types#EbsBlockDevice